ebean-core/src/main/java/io/ebeaninternal/server/bind/DataBind.java

```diff
@@ -117,36 +126,43 @@ public final PreparedStatement getPstmt() {
   @Override
   public void setString(String value) throws SQLException {
     pstmt.setString(++pos, value);
+    lastObject = value;
   }
 
   @Override
   public final void setInt(int value) throws SQLException {
     pstmt.setInt(++pos, value);
+    lastObject = value;
```

Comment:
I think this should instead be `lastObject = null` - avoid the boxing + this should never be validated

I think we should do this on all the primitive types + dates + timestamps + BigDecimal - all the types we are not interested in validating.